lib/requester.js

```diff
@@ -140,6 +171,32 @@ function challengeRequest(operation, requester) {
   request1.on('error', operation.errorListener);
   request1.end();
 }
+function credentialsRequest(operation, requester) {
+  kerberos = new Kerberos();
```

Comment:
Is it necessary to initialize the kerberos variable any time that credentialsRequest() is executed? Probably so, but if not, maybe it would be better to initialize only if the variable is null.